CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.89.1
+
+### Bug fixes
+
+* @PrimaryKey + @Required creates appropriate type cast when a primary key field type is `String` (#2653).
```

Comment:
Instead if would mention what hit the user so "@PrimaryKey + @Required on a String primary key no longer throws when using `copyToRealm` or copyToRealmOrUpdate (#2653)"